content/blog/2023/volunteering.md

```diff
@@ -0,0 +1,160 @@
+- Shadow: Jahnavi J
+- Zulip Stream: https://pyconindia.zulipchat.com/#narrow/stream/246476-wg.2Fcontent
+
+### Sponsorship
+
+- Chair: [Snehith Allamraju](https://pyconindia.zulipchat.com/#narrow/pm-with/609119-Snehith-Allamraju)
+- Shadow: [Abhishek Mishra](https://pyconindia.zulipchat.com/#narrow/pm-with/607971-Abhishek-mishra)
+
+### R&D (brainstorm new ideas like: poster presentation, digital swags)
+
+- Chair: [Bhanu K](https://pyconindia.zulipchat.com/#narrow/pm-with/335077-Bhanu-K)
+- Shadow: Pradhvan Bisht
+
+### Finance & Tickets
+
+- Chair: [Ramanathan Ramakrishnamoorthy](https://pyconindia.zulipchat.com/#narrow/pm-with/231811-Ramanathan-R)
+
+
+### Code of Conduct
+
+- Chair: [Bhanu K](https://pyconindia.zulipchat.com/#narrow/pm-with/335077-Bhanu-K)
+
+
+There are no specific requirements for volunteers. Contributions that enhance any area of the conference are welcome. If you need assistance with your work, feel free to reach out to any of us at our [Zulip channel](https://pyconindia.zulipchat.com/).
+
+Workgroup leaders and volunteers hold weekly meetings to discuss progress and ensure everyone is on the same page.
+
+## Getting Started with Volunteering
+
+Now that you understand what it means to volunteer and the impact you can make, follow these steps to get started:
+
+1. Fill out our [Volunteer form](https://docs.google.com/forms/d/e/1FAIpQLSdmlVnxDBENiXs17hQeUtgqOZdcsWI7lmQpXq5m9K-e0eS52Q/viewform) to let our team know you're interested in helping out. The leads from your chosen workgroup will reach out to you.
+
+2. Join the [mailing list](https://mail.python.org/mailman/listinfo/inpycon) to stay updated on all conference happenings, discussions, and progress.
+
+3. Join our [Zulip channel](https://pyconindia.zulipchat.com/) and follow the workgroups you'd like to contribute to. This is where you can connect with other volunteers, discuss ideas, and share your work.
+
+4. Proactively "Watch" the following GitHub repositories to stay up-to-date with task progress and contribute where you can:
+    - [Conference task tracking](https://github.com/pythonindia/inpycon2023-tasks)
+    - [PyCon 2023 Website](https://github.com/pythonindia/inpycon2023)
+    - [PyCon Blog](https://github.com/pythonindia/inpycon-blog)
+    - Our CFP software, [Junction](https://github.com/pythonindia/junction)
+
+
+## Weekly Check-in Call Details, feel free to join
+
+- Weekly check-in calls happen every week online on Thursday at 8:00 pm IST.
+- Download the calendar invite [here](https://drive.google.com/file/d/1ZCv0SWPJ0fklxnpnzidX7TK4U4SMbSe_/view?usp=sharing).
+- Join the call via Jitsi using the following link: [Jitsi Link](https://meet.jit.si/pyconindia2023-weekly)
+
+## Additional Volunteering FAQs
+
+1. Where can I contribute?
+    - Review the list of Zulip workstreams for different WorkGroups (WGs) [here](https://pyconindia.zulipchat.com/#narrow/stream/301938-WG-.2F-Team-Names).
```

Comment:
"This stream does not exist or is private."

Reply:
Fixed it. Thanks.